### PR TITLE
Introduce method to expose cache expiration

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

## Related Issues and Dependencies

Related-To: https://github.com/thoth-station/metrics-exporter/issues/615

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Add method to expose configured cache configuration.

### CODE REVIEW

File: thoth/common/exceptions.py [hunk 1]
```diff
@@ -17,29 +17,29 @@
 """Exceptions used within thoth-common package."""
 
 
-class ThothCommonException(Exception):
+class ThothCommonExceptionError(Exception):
```
Comment (harshad16) [inline thoth/common/exceptions.py:20]: why were the name changed ThothCommonExceptionError ? 
Exception in code itself is error
,Now,
right ?😅

Comment (pacospace) [reply thoth/common/exceptions.py:20]: https://prow.operate-first.cloud/view/s3/ci-prow-artifacts--e2081a4d-fd23-4347-93d5-f33e0604d95d/prow-logs/pr-logs/pull/thoth-station_common/1186/pre-commit/1413135455687282688 pre-commit is not happy apparentl :/

Comment (harshad16) [reply thoth/common/exceptions.py:20]: ah cool thnaks for poiniting this out.
```
flake8...................................................................Failed
- hook id: flake8
- exit code: 1
thoth/common/exceptions.py:20:8: N818 exception name 'ThothCommonException' should be named with an Error suffix
thoth/common/exceptions.py:24:8: N818 exception name 'NotFoundException' should be named with an Error suffix
thoth/common/exceptions.py:36:8: N818 exception name 'NotKnownThothIntegration' should be named with an Error suffix
thoth/common/exceptions.py:40:8: N818 exception name 'KebechetInputsMissing' should be named with an Error suffix
```

**Review by harshad16 — APPROVED** [review]
/lgtm
thanks :100:

**Pull request merged**